Add more references to klibs.io (#743)
JetBrains/kotlin-multiplatform-dev-docs: topics/overview/kmp-overview.md

  * Learn how KMP [compiles code](multiplatform-configure-compilations.md) and [builds binaries](multiplatform-build-native-binaries.md)
    for various targets.
  * Read about [publishing a multiplatform app](multiplatform-publish-apps.md)
    or a [multiplatform library](multiplatform-publish-lib-setup.md).

## Explore the Kotlin Mutliplatform library ecosystem

Thousands of multiplatform libraries are available for networking, storage, dependency injection, testing, UI, serialization, and more.

> Browse them on [klibs.io](https://klibs.io), JetBrains' search platform for Kotlin Multiplatform libraries.

Reviewer: ```suggestion
Browse them on [klibs.io](https://klibs.io/), a search platform maintained by JetBrains.
```
Same as above.
